fdbserver/TLogServer.actor.cpp

```diff
@@ -366,9 +366,9 @@ struct TLogData : NonCopyable {
 	                          // the set and for callers that unset will
 	                          // be able to match it up
 	std::string dataFolder; // folder where data is stored
-	Reference<AsyncVar<bool>> degraded;
```

Comment:
The old comments was not precise as this `degraded` is not for snapshots, move it out